src/index.ejs

```diff
@@ -1,5 +1,5 @@
 <!DOCTYPE html>
-<html lang="en">
+<html lang="{{.Locale}}">
```

Comment:
Yop @enguerran it's this PR which update index.ejs 😜  but you are right it's finally not a problem, sorry for this alert.